README.md

```diff
@@ -2,7 +2,135 @@
+- `fastq_1`: A URI (ex: a file path or web address) to either single-end FASTQ-formatted reads or one pair of pair-end FASTQ-formatted reads.
+- `fastq_2`: (Optional) If `fastq_1` is paired-end, then this field is a URI to reads that are the other pair of reads associated with `fastq_1`.
+- `assembly`: (Optional) A URI to an assembly associated with the sample, so that it may be referenced on the command line by the sample identifier for use as the reference for the whole pipeline. However, it may be easier to leave these fields blank and specify the reference using the `--refgenome` parameter.
+
+The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Please see [assets/samplesheet.csv](assets/samplesheet.csv) to see an example of a samplesheet for this pipeline.
+
+# Parameters
+
+## Mandatory
+
+The mandatory parameters are as follows:
+
+- `--input`: a URI to the samplesheet
+- `--output`: the directory for pipeline output
+
+## Optional
+
+The optional parameters are as follows:
+
+### Reference
+
+- `--refgenome`: a URI to the reference genome to use during pipeline analysis
+- `--reference_sample_id`: the sample identifier of a sample in the samplesheet that contains a provided `assembly` to use as a reference genome during pipeline analysis
+
+Please use only one of `--refgenome` or `--reference_sample_id` and not both.
+
+### SNVPhyl Parameters
+
+- `--window_size`: The window size for determining whether a region is high density.
+- `--density_threshold`: The minimum number of SNVs within the window size for a region to be considered high density.
+- `--min_coverage_depth`: The minimum depth of coverage for a position within the genome to pass the mapping quality check and be included in the analysis.
+- `--min_mapping_percent_cov`: The total percentage of positions within the genome that must have a depth of coverage greater than the minimum depth of coverage specified in order to pass the mapping quality check and be included in the analysis.
+- `--min_mean_mapping_quality`: The minimum mean mapping quality score for all reads in a pileup to be included in the analysis.
+- `--snv_abundance_ratio`: The proportion of reads required to support a variant to be included in the analysis.
+- `--min_repeat_length`: The minimum length when identifying repeats.
+- `--min_repeat_pid`: The minimum percent identity when identifying repeats.
+- `--skip_density_filter`: Whether or not to skip filtering low SNV density regions.
+
+Please refer to the [SNVPhyl documentation](https://snvphyl.readthedocs.io/en/latest/) for more detailed information about pipeline parameters.
+
+# Running
+
+## Test Data
+
+In order to run the pipeline with provided data, please run:
+
+```
+nextflow run main.nf -profile singularity --input https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv --refgenome https://raw.githubusercontent.com/phac-nml/snvphylnfc/main/assets/reference.fasta --outdir results
```

Comment:
Use `phac-nml/snvphylnfc` instead of `main.nf` for how to run pipeline.

Reply:
Updated in f9d87fcfb004ec10792bdce136ab155ccba31fd3